UnitsNet/CustomCode/Quantities/AreaMomentOfInertia.extra.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System;
+
+namespace UnitsNet
+{
+    public partial struct AreaMomentOfInertia
+    {
+        /// <summary>Get <see cref="Volume"/> from <see cref="AreaMomentOfInertia"/> divided by <see cref="Length"/>.</summary>
+        public static Volume operator /(AreaMomentOfInertia areaMomentOfInertia, Length length)
```

Comment:
Can you add a test case for this in 
https://github.com/angularsen/UnitsNet/blob/master/UnitsNet.Tests/CustomCode/AreaMomentOfInertiaTests.cs#L28-L41 ?

See other classes in the same CustomCode folder for examples, such as this one:
https://github.com/angularsen/UnitsNet/blob/master/UnitsNet.Tests/CustomCode/LengthTests.cs#L111-L116

Reply:
Test case and commit added! 😎